src/Microsoft.Identity.Client/AppConfig/ApplicationConfiguration.cs

```diff
@@ -40,27 +41,47 @@ internal ApplicationConfiguration()
         {
         }
 
-        public IHttpManager HttpManager { get; set; }
+        public bool IsBrokerEnabled { get; internal set; }
```

Comment:
= true (in MSAL.NET)

Reply:
not until we actually have a broker, which we don't.  i was just adding this into the config so we have it, enabling the broker will be a step for later when we build that feature in.

---
In reply to: [246261179](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/761#discussion_r246261179) [](ancestors = 246261179)